test/Twig/Tests/Extension/HumanReadableBytesTest.php

```diff
@@ -0,0 +1,113 @@
+<?php
+
+require_once __DIR__.'../../../../lib/Twig/Extensions/Extension/HumanReadableBytes.php';
+
+class HumanReadableBytesTest extends \PHPUnit_Framework_TestCase
+{
+    protected $humanReadableBytesExtension;
+
+    public function __construct()
+    {
+        parent::__construct();
+
+        $this->humanReadableBytesExtension = new HumanReadableBytes();
```

Comment:
this should be done in the ``setUp`` method, not in the constructor. Overwritting the constructor is not a supported extension point for ``\PHPUnit_Framework_TestCase`` and might create weird things